docs/pages/access-controls/idps/saml.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Using Teleport as a SAML identity provider
+description: How to configure and use Teleport as a SAML identity provider.
+h1: Teleport as a SAML identity provider
+---
+
+## How to use Teleport as an identity provider
+
+This guide will cover how to use Teleport as an identity provider so that Teleport
+can be used to authenticate to external services. 
+
+## Prerequisites
+
+- Teleport role with access to maintaining `saml_idp_service_provider` resources.  This is available in the default `editor` role.
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Configuring the SAML identity provider service
```

Comment:
Can you follow the same pattern as other guides i.e. using step numbers.

Step 1/3. Configure SAML IdP service
Step 2/3. Establish trust relationship ...

Reply:
Also, if configuration is not necessary, why do we need this step?

Reply:
I was more or less trying to emulate the other `sso` guides. :-) I can remove this.